MyRefreshDllsForTestRun.cmd

```diff
@@ -0,0 +1,17 @@
+SET WORKINGDIR=%~dp0
+SET _TargetLocation=%1
+SET _BuildConfiguration=%2
+IF [%_BuildConfiguration%] NEQ []  GOTO Start
+SET _BuildConfiguration=Debug
+
+:Start
+SET _PerfTestSourceLocation="%WORKINGDIR%\test\Microsoft.SqlTools.ServiceLayer.PerfTests\bin\%_BuildConfiguration%\netcoreapp1.0\win7-x64\publish"
+SET _ServiceSourceLocation="%WORKINGDIR%\src\Microsoft.SqlTools.ServiceLayer\bin\%_BuildConfiguration%\netcoreapp1.0\win7-x64\publish"
+
+
+
+dotnet publish %WORKINGDIR%\test\Microsoft.SqlTools.ServiceLayer.PerfTests -c %_BuildConfiguration%
+dotnet publish %WORKINGDIR%\src\Microsoft.SqlTools.ServiceLayer -c %_BuildConfiguration%
+		
+XCOPY /i /E /y %_PerfTestSourceLocation% "%_TargetLocation%\Tests"
+XCOPY /i /E /y %_ServiceSourceLocation% "C:\Users\wzhou\Source\external\vscode-mssql\sqltoolsservice\0.3.0-alpha.9\Windows" 
```

Comment:
Sorry, I was in a panic mode when I saw you were reviewing the partial commits. Don't want to waste your time. Will remove that